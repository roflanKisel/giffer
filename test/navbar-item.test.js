import React from 'react';
import { shallow } from 'enzyme';
import NavbarItem from '../components/navbar-item';

describe('NavbarItem', () => {
  it('should renders correctly', () => {
    const component = shallow(
      <NavbarItem name="Button" link="/" />
    );

    expect(component).toMatchSnapshot();
  });

  it('should render the DUMB component', () => {
    const wrapper = shallow(<NavbarItem name="Button" link="/" />);

    expect(wrapper.length).toEqual(1);
  });
});
