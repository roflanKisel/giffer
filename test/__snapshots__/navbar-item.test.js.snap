// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavbarItem should renders correctly 1`] = `
<NavbarItem
  link="/"
  name="Button"
/>
`;
